database/migrations/2017_03_31_211222_create_feedback_tables.php

```diff
@@ -151,7 +151,7 @@ public function up()
               'type_id' => '4',
               'form_id' => '2',
               'slug' => 'professional4',
-              'question' => 'They were professional and well delivered',
+              'question' => 'They service they provided was professional and well delivered.',
```

Comment:
Agreed it does not make sense at the moment but as pilot feedback will be disabled on launch and we don't have a question to replace it with, I'd suggest we just deploy with this question "hidden in the back end".